src/NServiceBus.Metrics/Extensions.cs

```diff
@@ -35,7 +35,7 @@ public static void ThrowIfSendonly(this FeatureConfigurationContext context)
         var isSendOnly = context.Settings.GetOrDefault<bool>("Endpoint.SendOnly");
         if (isSendOnly)
         {
-            throw new Exception("Windows performance counters are not supported on send only endpoints.");
+            throw new Exception("Metrics are not supported on send only endpoints.");
```

Comment:
I know Daniel only changed the text but why do we actually restrict the metrics to full endpoints?